fe/fe-core/src/main/java/com/starrocks/journal/bdbje/BDBEnvironment.java

```diff
@@ -305,6 +307,7 @@ protected void setupEnvironment() throws JournalException, InterruptedException
                     refreshLog((InsufficientLogException) e);
                 }
                 close();
+                closing = false;
```

Comment:
the closing=false has been set in the close(), why set it again here?

Reply:
my mistake. fix it.